person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknown')
+    @id = id
```

Comment:
- The id here should have a value @id = `Random.rand(1..100)`. Please, update this.